DEVELOPMENT.md

```diff
@@ -1,13 +1,15 @@
-# Build instructions
+# Developing for GoIV
 
-- Download and install Android studio (https://github.com/farkam135/GoIV/issues/60).
-- Install the SDK for Android 7.0 (API version 24) from within android studio (press the SDK manager button in the toolbar, an android face above a box with a down arrow).
-- Open the project.
-- Install lombok-intellij-plugin (https://github.com/mplushnikov/lombok-intellij-plugin)
-- If you get errors because modules are not found, use the SDK manager to add the missing modules.
+## Setting Up
+* Download and install [Git] (https://git-scm.com/)
+* Set up [Git with GitHub] (https://help.github.com/articles/set-up-git/)
+* Download and install [Android Studio] (https://developer.android.com/studio/index.html)
+* Set up [Android Development Environment] (https://spring.io/guides/gs/android/)
+* Install [Lombok IntelliJ Plugin] (https://github.com/mplushnikov/lombok-intellij-plugin#installation)
+* Set up Git on Android Studio (Configure > Settings > Version Control > Git)
```

Comment:
I wouldn't say essential but just recommended. Following the steps would reduce the errors. The lombok plugin is tricky, some are building fine without it, but some are having issues like in #253. But because of lombok plugin, you'll get errors for not enabling annotation processing.

Still, whatever works for you, as long as it compiles.